schemes.yaml

```diff
@@ -1007,6 +1007,32 @@ schemes:
       cyan: '0xb4f9f8'
       white: '0x5f8787'
 
+  nightfox: &nightfox
+    primary:
+      background: '0x192330'
+      foreground: '0xcdcecf'
+    normal:
+      black:   '0x393b44'
+      red:     '0xc94f6d'
+      green:   '0x81b29a'
+      yellow:  '0xdbc074'
+      blue:    '0x719cd6'
+      magenta: '0x9d79d6'
+      cyan:    '0x63cdcf'
+      white:   '0xdfdfe0'
+    bright:
+      black:   '0x575860'
+      red:     '0xd16983'
+      green:   '0x8ebaa4'
+      yellow:  '0xe0c989'
+      blue:    '0x86abdc'
+      magenta: '0xbaa1e2'
+      cyan:    '0x7ad5d6'
+      white:   '0xe4e4e5'
+    indexed_colors:
+      - { index: 16, color: '0xf4a261' }
+      - { index: 17, color: '0xd67ad2' }
```

Comment:
This seems rather unusual. What motivation is behind overwriting these indexed colors?

Reply:
Oh, that isn't my doing. I copied from the alacrity color scheme from this [file](https://github.com/EdenEast/nightfox.nvim/blob/main/extra/nightfox/nightfox_alacritty.yml) provided by the nightfox repo. **I'm not sure what the motivation is behind it is**, as alacritty looks the same with or without indexed colors. I noticed that other themes contained lines to alter those index colors as well so I did not think much of it.

Reply:
Unless you have a good reason for it, I'd rather remove it.

Reply:
Understood, the three lines have been removed.